app/views/decks/index.html.erb

```diff
@@ -1,4 +1,76 @@
-<%= link_to 'Create Deck', new_deck_path %><br>
-<% @decks.each do |deck| %>
-  <%= render deck %>
-<% end %>
+<div class="page page-dashboard">
+  <div class="wrapper wrapper--light">
+    <div class="container">
+      <div class="pagehead">
+        <div class="row">
+          <div class="col-sm-6">
+            <ol class="breadcrumb">
+              <li class="active">Decks</li>
+            </ol>
+          </div>
+          <div class="col-sm-6">
+            <div class="btn-group pull-right">
+              <%= link_to 'Create Deck', new_deck_path, class: 'btn btn-brand' %>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="container">
+
+    <%= render 'deck_summary', resource: @decks %>
```

Comment:
you had asked about naming the local variables `resource` in partials. it looks fine from the `render` call, however when you get in to the partial's code, having a generic variable name like `resource` isn't all that helpful. might be better to start naming local variables something more descriptive, like in this case
```ruby
decks: @decks
```